libs/domains/organizations/feature/src/lib/git-repository-setting/git-repository-setting.tsx

```diff
@@ -0,0 +1,85 @@
+import { type GitProviderEnum } from 'qovery-typescript-axios'
+import { Controller, useFormContext } from 'react-hook-form'
+import { useParams } from 'react-router-dom'
+import { InputSelect, LoaderSpinner } from '@qovery/shared/ui'
+import { upperCaseFirstLetter } from '@qovery/shared/util-js'
+import { getGitTokenValue } from '../git-provider-setting/git-provider-setting'
+import useRepositories from '../hooks/use-repositories/use-repositories'
+
+export interface GitRepositorySettingProps {
+  gitProvider: GitProviderEnum
+  disabled?: boolean
+}
+
+export function GitRepositorySetting({ disabled, gitProvider }: GitRepositorySettingProps) {
+  const { control, setValue, watch } = useFormContext()
+  const { organizationId = '' } = useParams()
+
+  const getGitToken = getGitTokenValue(gitProvider)
```

Comment:
You only fix `git-branch-settings`

Reply:
https://github.com/Qovery/console/pull/913/commits/930209f4f292c911acb87bf6dada6560cea35ae1